t a/tools/ci_build/build.py b/tools/ci_build/build.py
index 79314f345..ec58a7642 100644
--- a/tools/ci_build/build.py
+++ b/tools/ci_build/build.py
@@ -753,7 +753,7 @@ def install_python_deps(numpy_version=""):
     dep_packages.append("sympy>=1.10")
     dep_packages.append("packaging")
     dep_packages.append("cerberus")
-    run_subprocess([sys.executable, "-m", "pip", "install"] + dep_packages)
+    # run_subprocess([sys.executable, "-m", "pip", "install"] + dep_packages)


 def setup_test_data(source_onnx_model_dir, dest_model_dir_name, build_dir, configs):
